test/main/modules.js

```diff
@@ -49,6 +49,30 @@ QUnit.test( "does not run before subsequent tests", function( assert ) {
 	assert.equal( this.beforeCount, 1, "beforeCount did not increase from last test" );
 } );
 
+QUnit.module( "before (skip)", {
+	before: function( assert ) {
+		assert.ok( true, "before hook ran" );
+
+		if ( typeof this.beforeCount === "undefined" ) {
+			this.beforeCount = 0;
+		}
+
+		this.beforeCount++;
+	}
+} );
+
+QUnit.skip( "first test in module is skipped" );
+
+QUnit.test( "runs before first unskipped test", function( assert ) {
+	assert.expect( 2 );
+	assert.equal( this.beforeCount, 1, "beforeCount should be one" );
```

Comment:
I was following the convention in a few other tests in this file like [this](https://github.com/qunitjs/qunit/blob/master/test/main/modules.js#L42) one. I'm happy either way, but want to double-check that you prefer terseness over consistency here -- let me know which way you prefer.